### ISSUE TITLE
Multipage PDF

### ISSUE THREAD

**Issue by muneeb787** [penkzhou/plugin_html_to_pdf#1]
I have not found any way to ma pdf of more than one page ??? can you please guide me in this regard using this package how can i make a multi-page pdf!!!

**Comment by penkzhou** [comment]
it should generate multi-page pdf automatic. you can make a long html page and export it into this lib. try it.

**Comment by muneeb787** [comment]
Thank you , but is there no way to set content on each page !! actually, i have an array of HTML and html content on each index want to display on next page !!!

**Comment by muneeb787** [comment]
![image](https://github.com/penkzhou/plugin_html_to_pdf/assets/133331694/a4713e21-ab2f-4995-85a9-c8e4fe344873)
I am also facing this issue while using this plugin, what will be the solution for this

**Comment by penkzhou** [comment]
> ![image](https://private-user-images.githubusercontent.com/133331694/301809009-a4713e21-ab2f-4995-85a9-c8e4fe344873.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzUxMTksIm5iZiI6MTcwNjg3NDgxOSwicGF0aCI6Ii8xMzMzMzE2OTQvMzAxODA5MDA5LWE0NzEzZTIxLWFiMmYtNDk5NS04NWE5LWM4ZTRmZTM0NDg3My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMTUzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZTg3NGY2OWNlMTRhZTY5M2EzYjg2MDUzNzZmNjgwNDdiOGFiMDU2OTY1YjY0NzkzNTUyMWFhNjFkYzQxNjU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4jExgIRAsmhnTj7r7zyeScQth1SJsSXlIHWywKAqnHM) I am also facing this issue while using this plugin, what will be the solution for this

Is this your project's gradle file?

**Comment by muneeb787** [comment]
> > ![image](https://private-user-images.githubusercontent.com/133331694/301809009-a4713e21-ab2f-4995-85a9-c8e4fe344873.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzUxMTksIm5iZiI6MTcwNjg3NDgxOSwicGF0aCI6Ii8xMzMzMzE2OTQvMzAxODA5MDA5LWE0NzEzZTIxLWFiMmYtNDk5NS04NWE5LWM4ZTRmZTM0NDg3My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMlQxMTUzMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZTg3NGY2OWNlMTRhZTY5M2EzYjg2MDUzNzZmNjgwNDdiOGFiMDU2OTY1YjY0NzkzNTUyMWFhNjFkYzQxNjU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4jExgIRAsmhnTj7r7zyeScQth1SJsSXlIHWywKAqnHM) I am also facing this issue while using this plugin, what will be the solution for this
> 
> Is this your project's gradle file?

no its error in console

**Comment by codecrypt2020** [comment]
> it should generate multi-page pdf automatic. you can make a long html page and export it into this lib. try it.

Hello Team. We are able to create pdf but in pdf pages which are generated, page break is sometimes in between text and more than that, margin padding is not being applied on pdf pages at all. Is there any option which we can use from "printPdfConfiguration" apart from printSize and printOrientation? Any help would be greatly appreciated

**Comment by apurva010** [comment]
Hi @codecrypt2020 
Did you find a solution to it? I am also facing this issue, where it breaks the page in between text or a table.

**Comment by muneeb787** [comment]
Yes, I did a workaround for that. I created multiple PDFs, one for each page, and then merged them into one. This solved my issue, and this method is working on PROD now.

**Comment by apurva010** [comment]
@muneeb787 Thank you for the response.
I’m interested in how you differentiated the HTML content and determined the page boundaries to generate separate PDFs.

**Comment by muneeb787** [comment]
I managed that at the code level, as I had the HTML array for Pages, and based on that, I generated separate PDFs and merged them at the end!
